README.md

```diff
@@ -1,28 +1,29 @@
 # Jenkins Prometheus Metrics Plugin
 
 ## About
-Jenkins Prometheus Plugin expose an endpoint (default `/prometheus`) with metrics where a Prometheus Server can scrape.
+Jenkins Prometheus Plugin exposes an endpoint (default `/prometheus`) with metrics where a Prometheus Server can scrape.
 
-Documentation can be found [here](https://plugins.jenkins.io/prometheus)
+Documentation can be found [here](https://plugins.jenkins.io/prometheus).
 
 Please note that the documentation is a WIP.
 
 ## Metrics exposed
-Currently only metrics from the [Metrics-plugin](https://github.com/jenkinsci/metrics-plugin) and summary of build
-duration of jobs and pipeline stages
+Currently the metrics are from the [Metrics-plugin](https://github.com/jenkinsci/metrics-plugin) with summary of build
+duration of jobs and pipeline stages.
 
 ## Environment variables
 
 `PROMETHEUS_NAMESPACE` Prefix of metric (Default: `default`).
 
-`PROMETHEUS_ENDPOINT` REST Endpoint (Default: `prometheus`)
+`PROMETHEUS_ENDPOINT` REST Endpoint (Default: `prometheus`).
 
-`COLLECTING_METRICS_PERIOD_IN_SECONDS` Async task period in seconds (Default: `120` seconds)
+`COLLECTING_METRICS_PERIOD_IN_SECONDS` Async task period in seconds (Default: `120` seconds).
 
 ## Building
 
-    mvn clean install
-    mvn hpi:hpi
+    `mvn clean install`
+
```

Comment:
Spaces like this are not needed

Reply:
I am working on it. Will it be convenient for you to assign the issue to me?

Reply:
Because you are not part of the org I cannot assign to you but this is your body of work 🙂

Reply:
Thanks! I am on it!